frontend/src/pages/Alerts/ErrorAlert/ErrorAlertPage.tsx

```diff
@@ -402,9 +392,19 @@ export const ErrorAlertPage = () => {
 							</Box>
 
 							<Form store={formStore} resetOnSubmit={false}>
-								<Stack gap="40">
+								<Stack gap="12">
 									<AlertTitleField />
-									<ErrorAlertForm />
+									<SearchContext
+										initialQuery={initialQuery}
+										onSubmit={handleUpdateInputQuery}
+									>
+										<ErrorAlertForm
+											hideRegexExpression={
+												isCreate ||
+												!alert?.RegexGroups?.length
```

Comment:
Only show regex expression for older alerts that have non empty regex expressions